web/src/pages/glossary/index.tsx

```diff
@@ -0,0 +1,15 @@
+import MetaHead from "@/components/meta/Head";
+import Title from "@/components/ui/Title";
+
+export default function GlossaryPage() {
+  return (
+    <>
+      <MetaHead
+        title="Glossary | ARC Services Portal"
```

Comment:
can we use tooltips instead of a separate page?

Reply:
we could do both and see if users like both or have a strong preference for one?